src/Option/README.md

```diff
@@ -0,0 +1,24 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `aria-selected` (boolean; optional):
+The ListBox component will handle the `aria-selected` attribute itself, but the `aria-selected` prop can be used as an override, if necessary.
+
+### `children` (element array; required)
+`children` specifies the list of elements that will be displayed within the Option component.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the Option component. The default is an empty array. (do I mention the existing classnames here?)
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered Option component. The default is div. Example alternative include li, Button, etc.
```

Comment:
I have found ListBox implementations that use only divs, but I have also found ones that use list elements, so I figured this would give more flexibility in terms of what Options can be. An Option could also theoretically be a Button, for example.